### PR TITLE
added reserved for connection info in user description tab

### PR DESCRIPTION
**Pull request by ct-goranpalibrk**
Signed-off-by: CT\pgoran <goran.palibrk@comtrade.com>

Brief description of the PR.
Added new field in Description tab in User view.

**Related Issue**
This PR fixes issue #1993 

**Description of the solution adopted**
A more detailed description of the changes made to solve/close one or more issues.
If the PR is simple and easy to inderstand this section can be skipped.

**Screenshots**
If applicable, add screenshots to help explain your solution

**Any side note on the changes made**
Description of any other change that has been made, which is not directly linked to the issue resolution
[e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #1993.

File: service/device/registry/api/src/main/java/org/eclipse/kapua/service/device/registry/connection/DeviceConnectionService.java [hunk 1]
```diff
@@ -71,4 +71,7 @@ void connect(DeviceConnectionCreator creator)
      */
     void disconnect(KapuaId scopeId, String clientId)
             throws KapuaException;
+
+    DeviceConnection findByUserId(KapuaId scopeId, KapuaId userId )
```
Comment (lorthirk) [inline service/device/registry/api/src/main/java/org/eclipse/kapua/service/device/registry/connection/DeviceConnectionService.java]: Please don't add this method. Add a `private` method in `GwtUserServiceImpl` that performs this query instead.

**Comment by Coduz** [comment]
HI @ct-goranpalibrk 

any news on this one?

Regards, 

_Alberto_

**Comment by ct-goranpalibrk** [comment]
Hi @Coduz I am trying Claudio's proposal right now.

**Comment by LeoNerdoG** [comment]
I have rechecked this and from GUI side it looks perfect.

**Comment by ct-goranpalibrk** [comment]
Hi @Coduz @lorthirk I madechanges, please take a look.

**Comment by LeoNerdoG** [comment]
I ahve checked again and on GUI is everything OK.

File: console/module/user/pom.xml [hunk 2]
```diff
@@ -42,6 +42,10 @@
             <groupId>org.eclipse.kapua</groupId>
             <artifactId>kapua-user-api</artifactId>
         </dependency>
+         <dependency>
```
Comment (Coduz) [inline console/module/user/pom.xml]: Please remove this dependency

Comment (ct-goranpalibrk) [reply console/module/user/pom.xml]: Hi @Coduz , as @lorthirk proposed me to make a new private method in `GwtUserServiceImpl`  , I had to add dependency that is needed for getting Devices from DeviceQuery. So If I remove this dependeny, this solution will not work.

Comment (Coduz) [reply console/module/user/pom.xml]: Ah right, I didn't see that! :D Sorry

**Comment by Coduz** [comment]
When merging please squash commits

**Comment by ct-goranpalibrk** [comment]
@Coduz Commits squashed.

**Pull request merged**